week-2/extended-project/banzai/init.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+rm -rf ~/todo_list
```

Comment:
Frank, please can you explain me the functionality of these lines?

Reply:
This command delete directory todo_list

Reply:
ok, thanks